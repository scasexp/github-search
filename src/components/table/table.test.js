import React from 'react';
import renderer from 'react-test-renderer';
import TableApp from './index';

test('MatchSnap', () => {
    const component = renderer.create(
      <TableApp list={[]}/>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

