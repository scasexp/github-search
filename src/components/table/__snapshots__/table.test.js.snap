// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MatchSnap 1`] = `
<table>
  <thead>
    <tr>
      <th>
        Nombre Repositorio
      </th>
      <th>
        Url
      </th>
    </tr>
  </thead>
  <tbody />
</table>
`;
